lib/components/programs-list/index.tsx

```diff
@@ -42,6 +46,7 @@ type ProgramListingProps = {
   date: string;
   time: string;
   registerLink: string;
```

Comment:
If we made the recordingLink optional, I think it would also make sense to make the registerLink optional. I'm thinking about: what if the admins of the future site want to post an event to let people know that it exists, but don't have the register link yet? What do you think? @yanDavid21

Reply:
Just checked with the Davids and registerLink doesn't need to be optional. If an event exists but doesn't have a register link yet, the event just wouldn't show up on the Programs page yet